checkBattery.h

```diff
@@ -0,0 +1,24 @@
+
+#define E_NOT_OK      0
+#define E_OK          1
+#define TEMP_MIN      0
+#define TEMP_MAX      45
+#define SOC_MIN       20
+#define SOC_MAX       80
+#define CHARGE_LIMIT  0.8
+
+//#define TEMPERATURE_IN_FARENHEIT
+
+#ifdef TEMPERATURE_IN_FARENHEIT
```

Comment:
Consequence of compiler directive: If we are interfacing with both types of sensors (those which give celsius and some others that give Fahrenheit), a single binary cannot do the job